jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -526,15 +557,27 @@ String getFormat() {
    * @return the configured extra layer file permissions
    */
   List<PermissionConfiguration> getExtraDirectoryPermissions() {
-    String property = getProperty(PropertyNames.EXTRA_DIRECTORY_PERMISSIONS);
+    String deprecatedProperty = getProperty(PropertyNames.EXTRA_DIRECTORY_PERMISSIONS);
+    String newProperty = getProperty(PropertyNames.EXTRA_DIRECTORIES_PERMISSIONS);
+    if (deprecatedProperty != null) {
+      getLog()
+          .warn(
+              "The property 'jib.extraDirectory.permissions' is deprecated; "
```

Comment:
Perhaps more maven-y message? and similar log warning when both properties used?

Reply:
Resolved (https://github.com/GoogleContainerTools/jib/pull/1626#discussion_r276320616 and
https://github.com/GoogleContainerTools/jib/pull/1626#discussion_r276321076)